src/main/kotlin/org/rust/lang/utils/RsDiagnostic.kt

```diff
@@ -1799,6 +1810,7 @@ sealed class RsDiagnostic(
         )
     }
 
+
```

Comment:
nit: the line seems to be added by accident

Reply:
I've reformatted the whole file to be inline with the code format style